src/kixi/uploader.clj

```diff
@@ -0,0 +1,53 @@
+(ns kixi.uploader
+  (:require [amazonica.aws.s3 :as s3]
+            [kixi.maws :refer [witan-admin-prod-creds]]
+            [clojure.java.io :as io]
+            [clojure.string :as string])
+  (:import [java.io ByteArrayInputStream File InputStream]))
+
+(def mfa "442962")
```

Comment:
It feels like an env var override might be good here? Obv this mfa isn't going to work

Reply:
It should be an arg to the upload fn really.